### ISSUE TITLE
Establish content and social media strategies

### ISSUE THREAD

**Issue by astrojuanlu** [kedro-org/kedro-devrel#10]
_(Drawing from some ideas by @stichbury)_

Kedro channels:

- Documentation
- QB Medium
- Blog (upcoming)

Possible third-party channels:

- Twitter
- Microblogging Fediverse (Mastodon or similar)
- LinkedIn
- Reddit
- Instagram
- YouTube
- Twitch
- TikTok

Guest channels:

- Towards Data Science
- ?

Obviously I was overly comprehensive here and I don't think we should strive to be everywhere, nor re-posting content across channels. Instead, we should pick a few of those channels based on

- Target audience/User personas
- Content format

For example: discarding Instagram and LinkedIn, picking Mastodon over Twitter, choosing YouTube over Twitch, holding on Reddit and TikTok for now. Hence long-form blogging + microblogging + occasional video for now. To be discussed.

**Comment by stichbury** [comment]
I'd like to suggest DZone (to syndicate content, with a canonical link back to wherever we publish it) and dev.to (likewise), where it makes sense to re-post to bump the ranking for organic search. 

Although DZone is not what it was, there are still merits and I've a relationship with the editorial team so can get content published (as long as it's not too promotional). Likewise dev.to, and we could set up our own channel.

I have a few ideas for additional guest channels but will drop them in the strategy doc I started a while ago.

**Comment by astrojuanlu** [comment]
Good suggestions. And there's a point to be made too about where do we want to keep a personal profile and push Kedro content and where do we want to have a "branded" account. To be discussed as well.

**Comment by astrojuanlu** [comment]
Heads up: the Linux Foundation seems to have started a Mastodon server https://social.lfx.dev/about (see for example https://social.lfx.dev/@deltalakeoss, https://social.lfx.dev/@pytorch).

**Comment by astrojuanlu** [comment]
Sent an email to the instance admin.

**Comment by amandakys** [comment]
I had a couple ideas to add

1. Slack: A slackbot to regularly push our 'Kedro Tips' or 'Did you know?' content to bring discoverability to features we want our users to use. Theres also no reason why these prompts can't be regularly shared on a variety of platforms like the mini updates scikit-learn share on linkedIn. (https://kedro-org.slack.com/archives/C048BAGLTGT/p1678358043712709)

2. Medium: a Kedro account where we posted tutorials/examples on how to do things in a step-by-step format i.e. How to convert a jupyter notebook to Kedro. This would serve as a complement to our documentation so we separate out "knowledge content" and "practical content" and provide more hands on, but specific examples on how to do things on Kedro. It would also allow us to respond more quickly to questions that pop up regularly, write an article about what to do, post it. This might also be a step towards decluttering our docs. 

The QB design guild have an account where we post about our design work and best practice to sort of spread the word about our design work (I'm following up on this with design about their strategy will update)

**Comment by stichbury** [comment]
> 2. Medium: a Kedro account where we posted tutorials/examples on how to do things in a step-by-step format i.e. How to convert a jupyter notebook to Kedro. This would serve as a complement to our documentation so we separate out "knowledge content" and "practical content" and provide more hands on, but specific examples on how to do things on Kedro. It would also allow us to respond more quickly to questions that pop up regularly, write an article about what to do, post it. This might also be a step towards decluttering our docs.
> 

Thanks for these great ideas @amandakys! I like the idea of a slackbot -- although I'd actually go a lot further and find a way to make one smart enough for a user to "pull" the answer to questions about Kedro with a set of useful info drawn from linen and the slack history, rather than "push" how-to. But in the more immediate world, a set of repeatable content is definitely high on the shopping list, and we can drop that via range of channels.

I think your Medium idea is sound...but we have the Kedro blog itself on the way this/next sprint, so that probably serves the purpose. Rather than spread ourselves across channels, I think I'd rather we try first to put out a range of content on one blog, with some syndication to others, such as DZone and dev.to. I also think dev.to is a nice way for third parties to write about Kedro, without having it go through our blog editorial processes, so if we find the community want to share written tips, I'd be tempted to use that as a third party channel. WDYT?

**Comment by astrojuanlu** [comment]
Dev.to instructions for organizations https://dev.to/organization-info

**Comment by astrojuanlu** [comment]
In terms of channels and platforms, for now we settled on: [Blog](https://kedro.org/blog) ([lots of stuff in the pipeline](https://github.com/kedro-org/kedro-devrel/labels/blog%20post)), [Dev.to](https://dev.to/kedro) (to repost content from our blog), [YouTube](https://www.youtube.com/@kedro-python) (trainings, talks, videos of any sort), [Mastodon](https://social.lfx.dev/@kedro) (news, updates, tips, threads), and possibly, hopefully, an email newsletter (periodic summaries of all the above). We're retiring our Meetup page.

Lots of potential for new ideas (like the "tip of the day/week" @amandakys suggested), but I think it's a good moment to close this issue, start saying "hello world" everywhere soon, and keep iterating from there.
